src/app/(views)/github/t/@pullButtons/pulls/page.tsx

```diff
@@ -29,6 +38,62 @@ export default function Page() {
     openAllRepo()
   }
 
+  const handleSelectAllDependabotClick = () => {
+    if (repos) {
+      repos.map((repo) => {
+        console.log(`c.log ## repo ##`, repo)
+        if (repo.pulls) {
+          repo.pulls.map((pull) => {
+            if (pull.user.login === 'jisung-lee7') {
```

Comment:
I would use const for this.

```typescript
if (pull.user.login === DEPENDABOT_USER_NAME) {
  // blah blah
}
```

Reply:
Oh cool

Reply:
You did not resolved this

Reply:
Ah~ 'const for this', okay. I missed it.